docs/workshop/ml_pipeline.md

```diff
@@ -1,6 +1,6 @@
 # Create an ML Pipeline using Elyra and Kubeflow Pipelines
 
```

Comment:
In this section, we will learn how to run the training pipeline of the ML model in automation.

To learn more about the training process refer to the chapter on [model development](./model_development.md).
```suggestion

```

Reply:
can you include the above to the introduction. We are going into creating the pipeline very abruptly